Genre content DB.sql

```diff
@@ -0,0 +1,77 @@
+CREATE DATABASE content; 
+USE content;
+
+CREATE TABLE Books ( 
+	Book_ID INTEGER PRIMARY KEY NOT NULL,
+	Book_Name VARCHAR(100) UNIQUE NOT NULL,
+    Book_Author VARCHAR(30),
+    Book_Genre VARCHAR(25),
+	Price FLOAT NOT NULL, 
+    Book_Script VARCHAR(1000) 
+    );
+
+CREATE TABLE Anime ( 
+	Anime_ID INTEGER PRIMARY KEY NOT NULL,
+	Anime_Name VARCHAR(50) UNIQUE NOT NULL,
+    Anime_Genre VARCHAR(25),
+	Where_TW VARCHAR(25), 
+    Anime_Script VARCHAR(1000) 
+    );
+
```

Comment:
Test

Reply:
Yeah I think at this point we dont really need to worry about age ratings, but mentioning the possibility of doing so in the documentation is a good idea